projects/libusb.info/package.yml

```diff
@@ -0,0 +1,82 @@
+distributable:
+  url: https://github.com/libusb/libusb/releases/download/v{{ version }}/libusb-{{ version }}.tar.bz2
+  strip-components: 1
+
+versions:
+  github: libusb/libusb/releases/tags 
+  strip: /^v/
```

Comment:
I would add an `ignore: /-rc/` to save us issues in the future.

Reply:
the version parser already rejects prereleases